plugin.audio.ampache/resources/lib/ampache_plugin.py

```diff
@@ -188,9 +177,9 @@ def addLinks(elem,object_type,useCacheArt,mode):
 
     for node in elem.iter(elem_type):
         cm = []
-        object_id = int(node.attrib["id"])
-        #xbmc.log("AmpachePlugin::addLinks: object_id  - " + str(object_id) , xbmc.LOGDEBUG )
-        #xbmc.log("AmpachePlugin::addLinks: node " + ET.tostring(node) , xbmc.LOGDEBUG )
+        object_id = node.attrib["id"]
+        if object_id is None or object_id == "":
```

Comment:
if not object_id: would cover both cases, identity and comparison

Reply:
I will test in the next release